ouroboros-network-framework/src/Ouroboros/Network/Driver/Simple.hs

```diff
@@ -0,0 +1,217 @@
+{-# LANGUAGE GADTs #-}
```

Comment:
Please use `git mv`.

Reply:
I did, but this module is actually being partially copied into two places, and this is the second one that git therefore does not track.

The rationale here is that the Simple driver is moved to the `typed-protocol-examples` package, where it is used both as an example, and can also be used in tests for other packages.

For the moment we still need the simple driver in the real code, but it'll soon be replaced by the driver with time and size limits. Same with the other byte limit driver that we currently only use for the handshake protocol, it'll also be replaced by the more general limits driver.